crates/sui/src/main.rs

```diff
@@ -45,13 +46,21 @@ async fn main() {
 
     let args = Args::parse();
     let _guard = match args.command {
-        SuiCommand::Console { .. }
-        | SuiCommand::Client { .. }
-        | SuiCommand::KeyTool { .. }
-        | SuiCommand::Move { .. } => telemetry_subscribers::TelemetryConfig::new()
-            .with_log_level("error")
-            .with_env()
-            .init(),
+        SuiCommand::Client {
```

Comment:
Is this intended to land with this change?

Reply:
Yes, without having the logging output during profiling and replay, it completely obscures why a command has failed